pyat/at/matching/globalfit.py

```diff
@@ -91,6 +95,7 @@ def fit_tune(ring: Lattice, refpts1: Refpts, refpts2: Refpts, newval,
         fit_integer: bool (default=False), use integer tune
         regex:      Using regular expressions for refpt string matching;
                     Default: False
+        delta: gradient variation applied to magnets. Default 1e-6 
```

Comment:
If I'm right, the default will be 1.e-5 for the tune fit: (`1.e-6 * 10 ** 1`)
and 1.e-4 for chromaticity (`1.e-6 * 10 ** 2`)